templates/ethernet_nmconnection.j2

```diff
@@ -25,6 +25,12 @@ mtu={{ item.mtu }}
 interface-name={{ item.device }}
 parent={{ item.device | regex_replace(vlan_interface_regex, '\g<interface>') }}
 id={{ item.device | regex_replace(vlan_interface_regex, '\g<vlan_id>') }}
+{% if item.ingress_priority_map is defined %}
+ingress-priority-map={{ item.ingress_priority_map }}
```

Comment:
It looks like there is an equivalent setting in network-scripts: https://people.freedesktop.org/~lkundrak/nm-docs/nm-settings-ifcfg-rh.html